sdcm/scylla_bench_thread.py

```diff
@@ -184,8 +190,11 @@ def run(self):
         self.executor = concurrent.futures.ThreadPoolExecutor(max_workers=self.max_workers)
 
         for loader_idx, loader in enumerate(loaders):
+            event = ScyllaBenchEvent(node=loader, stress_cmd=self.stress_cmd)
```

Comment:
It would be better to create this event inside the run and return it from it

Reply:
Changed